app/components/dropdown-widget/component.js

```diff
@@ -172,47 +177,64 @@ export default Ember.Component.extend({
             }
 
             let filter_data = await Ember.$.ajax({
-                url: 'https://dev-labs.cos.io/api/records/search/creativeworks/_search?request_cache=true',
+                url: 'https://dev-labs.cos.io/records/_search?request_cache=true',
```

Comment:
is this url in ENV? if so we should use it from there.

Reply:
fixed